covigator/dashboard/tabs/lineages.py

```diff
@@ -84,6 +85,14 @@ def get_lineages_tab_left_bar(queries: Queries, data_source: DataSource):
                 multi=True
             ),
             html.Br(),
+            dcc.Markdown("""Select time period for smoothing"""),
+            dcc.Dropdown(
+                id=ID_DROPDOWN_PERIOD,
+                options=[{'label': c, 'value': c} for c in range(1, 32)] + [{'label': 'Disable', 'value': False}],
```

Comment:
I suggest you set the label to `"{} days".format(c)` so it is more explicit what the numbers correspond to.